src/pipecat/services/cartesia.py

```diff
@@ -386,14 +385,17 @@ async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
                 _experimental_voice_controls=voice_controls,
             )
 
+            await self.start_tts_usage_metrics(text)
```

Comment:
Just to be sure, I see that we are invoking `start_tts_usage_metrics`, but I don't see us invoking anything to stop these metrics. Does this happen inside `stop_ttfb_metrics`, or is it something we are missing or not needed?

Reply:
Yes, `start_tts_usage_metrics` doesn't have a stop counterpart.